.github/actions/integration-tests/action.yml

```diff
@@ -0,0 +1,20 @@
+name: 'Integration tests'
+description: 'Run integration tests'
+inputs:
+  github-token:
+    description: 'The GITHUB_TOKEN secret'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+    - name: Integration Tests
+      if: always()
+      run: npm run test:integration -- --reporter json --reporter-option 'output=reports/integration-tests.json'
```

Comment:
I think you spoke to this already on a prior PR (apologies if so), but were we seeing issues with the env var route for these mocha values or do you just think it's better with inline args?

I've no preference either way and am content with whatever works, more just curious

Reply:
Don't think it has come up in any of the previous PRs. Not sure whether we pass the options as args or env vars makes a huge amount of difference either way tbh.